src/extension.ts

```diff
@@ -359,6 +361,13 @@ export function activate(context: VSCode.ExtensionContext) {
   installClasspathListener(languageClient);
 }
 
+async function showNotificationForFirstSecretsIssue(context: VSCode.ExtensionContext) {
+  VSCode.window.showWarningMessage("SonarLint detected some secrets in one of the open files.\n" +
+      "We strongly advise you to review those secrets and ensure they are not committed into repositories. " +
+      "Please refer to the SonarLint tool window for more information.");
```

Comment:
There is no SonarLint tool window in VSCode. Would it be possible to add an action to focus on the "Problems" view instead?

Reply:
Done.